packages/server/computation_container/test/unit_test/ctodb_test.cpp

```diff
@@ -33,7 +33,7 @@ TEST(ComputationToDbTest, SuccessReadShareTest)
     std::vector<std::vector<std::string>> true_table = {{"1", "2"}, {"3", "4"}};
     std::vector<std::string> true_schema = {"attr1", "attr2"};
     EXPECT_EQ(true_table, read_data.getTable());
-    EXPECT_EQ(true_schema, read_data.getSchemas());
+    EXPECT_EQ(true_schema, read_data.getSchemaNames());
```

Comment:
I thought it would be good to do the same as the test added this time (see below).
I was thinking that it would be better not to use `getSchemaNames`.

https://github.com/acompany-develop/QuickMPC/blob/bbefeb8248ce0164ffcfe44f2ae7a451bf617644/packages/server/computation_container/test/unit_test/ctodb_test.cpp#L101-L105

Reply:
@mdonaka 
Indeed, noted with thanks!  
I will fix it.